### DISCUSSION TITLE
Update imported worksheet

### DISCUSSION THREAD

**Discussion by dwchan69** [dfinke/ImportExcel#1575] [Q&A]
I have an XLS with multiple worksheets, one of which is name OS.  It is a simple 2 columns by 6 rows.  each row consist of a name of an OS (i.e. Win10, Win11, Win2022, Win2019, etc) and a Build status if it needs to be built ($true or $false)

I read in the data simple with this 
$OSParameters = Import-Excel -Path $DataSourcePath -WorksheetName 'OS'

I can change the Build status in the array value $OSParameters as needed to keep track of what has been built, etc, as long as the PowerShell is going through its iterate loop. 

However, if I want to save it back to the original file, with the following Export-Excel syntax, 

Export-Excel -Path G:\excel.xlsx -WorksheetName OS -InputObject $OSParameters  -ClearSheet

the file would open by with an error warning, and the workseet 'OS' is deleted.  Any though on this?

**Comment by dfinke** [comment]
Would need to see a simpler repro of this. XLSX, script etc.
